src/main/java/rx/observables/BlockingObservable.java

```diff
@@ -477,4 +473,227 @@ private void awaitForComplete(CountDownLatch latch, Subscription subscription) {
             throw new RuntimeException("Interrupted while waiting for subscription to complete.", e);
         }
     }
+    
+    /**
+     * Runs the source observable to a terminal event, ignoring any values and rethrowing any exception.
+     */
+    @Experimental
+    public void run() {
```

Comment:
There is already a method named `subscribe()`. I've put in this alias for discoverability.